app/SearchApi/SearchApi.Web/Notifications/WebHookNotifierPersonSearchAccepted.cs

```diff
@@ -0,0 +1,88 @@
+﻿using AutoMapper;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using SearchApi.Core.Adapters.Contracts;
+using SearchApi.Web.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SearchApi.Web.Notifications
+{
+
+    public class WebHookNotifierPersonSearchAccepted : BaseApiNotifier, ISearchApiNotifier<PersonSearchAccepted>
+    {
+
+        private readonly HttpClient _httpClient;
+        private readonly SearchApiOptions _searchApiOptions;
+        private readonly ILogger<WebHookNotifierPersonSearchAccepted> _logger;
+        private readonly IMapper _mapper;
+
+        public WebHookNotifierPersonSearchAccepted(HttpClient httpClient, IOptions<SearchApiOptions> searchApiOptions,
+            ILogger<WebHookNotifierPersonSearchAccepted> logger, IMapper mapper)
+        {
+            _httpClient = httpClient;
+            _logger = logger;
+            _searchApiOptions = searchApiOptions.Value;
+            _mapper = mapper;
+        }
+
+        public async Task NotifyEventAsync(Guid searchRequestId, PersonSearchAccepted personSearchAccepted,
```

Comment:
We could have autofac imapper injected in the consumer, then this method could take the providerSearchEvent as a parameter, then you can reuse the same method for notifying any events, so the service could implement `ISearchApiNotifier<ProviderSearchEventStatus>`

Reply:
Resolved, was still refactoring.. should be in now